WordPress/Classes/Services/NotificationFilteringService.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+/// The service is created to support disabling WordPress notifications when Jetpack app is installed
+/// This is a temporary solution to avoid duplicate notifications during the migration process from WordPress to Jetpack app
+/// This service and its usage can be deleted once the migration is done
+final class NotificationFilteringService {
+    private var notificationSettingsLoader: NotificationSettingsLoader
+    private var notificationsEnabled: Bool = false
+    private let allowDisablingWPNotifications: Bool
+    private let isWordPress: Bool
+    private let userDefaults = UserDefaults(suiteName: WPAppGroupName)
```

Comment:
Perhaps we should add a comment here that this class and solution uses App Groups which allows JetPack app to set the state of wordPressNotificationsEnabled since both WordPress and JetPack are using the same AppGroup name?  While testing + reviewing the solution it took me awhile to understand what was going on until I saw WPAppGroupName

Reply:
I agree. Adding it to the description. 👍